packages/scripts/index.js

```diff
@@ -0,0 +1,117 @@
+    ? projectRollupConfigFilePath
+    : path.resolve(path.join(__dirname), 'basePluginRollupConfig.js');
+};
+
+const OUTPUT_DIR_REGEX = /\/?(\w+)$/;
+
+const getOutputDir = (output) => {
+  const outputDirMatch = OUTPUT_DIR_REGEX.exec(output);
+  return outputDirMatch.length > 1 ? outputDirMatch[1] : output;
+};
+
+const startHandler = ({ file }) => {
+  process.env.ROLLUP_WATCH = true;
+
+  loadConfigFile(getRollupConfigPath(file))
+    .then(async ({ options, warnings }) => {
+      const spinners = {};
+      const watcher = rollup.watch(options);
+
+      warnings.flush();
+      watcher.on('event', (event) => {
+        switch (event.code) {
+          case 'BUNDLE_START':
+            event.output.forEach((output) => {
+              const outputDir = getOutputDir(output);
+              const spinner = ora({
+                text: chalk.blue.bold(`${event.input} -> ${outputDir}`),
+              });
+              spinners[`${event.input}-${output}`] = spinner;
+
+              spinner.start();
+            });
+            break;
+          case 'BUNDLE_END':
+            event.output.forEach((output) => {
+              const spinner = spinners[`${event.input}-${output}`];
+              if (spinner) {
+                const outputDir = getOutputDir(output);
+                spinner.succeed(chalk.green.bold(`${event.input} -> ${outputDir}`));
+                delete spinners[`${event.input}-${output}`];
+              }
+            });
+            break;
+          case 'ERROR':
+            Object.entries(spinners).forEach(([key, spinner]) => {
+              spinner.fail(event.error);
+              delete spinners[key];
+            });
+            break;
+        }
+      });
+
+      watcher.close();
+    })
+    .catch((err) => console.error(err));
+};
+
+const printBundleStart = (option) => {
+  let message = option.output.map((output) => `${option.input} -> ${output.dir}...`).join('\n');
+  console.log(chalk.blue.bold(message));
+};
+const printBundleComplete = (option, completedTimeInMS) => {
+  const outputFolder = chalk.green.bold(option.output.map((output) => output.dir).join(', '));
+  const completedTimeInS = chalk.green.bold(`${completedTimeInMS / 1000}s`);
+  console.log(chalk.green(`created ${outputFolder} in ${completedTimeInS}`));
+};
+
+const buildHandler = ({ file }) => {
+  loadConfigFile(getRollupConfigPath(file))
+    .then(async ({ options, warnings }) => {
+      warnings.flush();
+
+      for (const o of options) {
+        const start = Date.now();
+        console.log('');
+        printBundleStart(o);
+
+        const bundle = await rollup.rollup(o);
+        await Promise.all(o.output.map(bundle.write));
+
+        printBundleComplete(o, Date.now() - start);
+      }
+    })
+    .catch((err) => console.error(err));
+};
+
+const optionsConfigurer = (yargs) =>
+  yargs.option('file', {
+    alias: '-f',
+  });
+
+require('yargs/yargs')(process.argv.slice(2))
```

Comment:
I've seen this before:
```
require('yargs/yargs')(process.argv.slice(2))
```


but I've always simply used:
```
require('yargs')
```
Whats the diff?

Reply:
Hmm, I can't remember exactly why, but I had issues with chaining `help` and `demandCommand` when done the other way. But I don't see that happening anymore. 🤷